Client/Frontend/Browser/TopTabCell.swift

```diff
@@ -65,10 +65,16 @@ class TopTabCell: UICollectionViewCell, ThemeApplicable, LegacyTabTrayCell, Reus
 
         titleText.text = tab.getTabTrayTitle()
         accessibilityLabel = getA11yTitleLabel(tab: tab)
+        showsLargeContentViewer = true
+        largeContentTitle = tab.getTabTrayTitle()
         isAccessibilityElement = true
 
         closeButton.accessibilityLabel = String(format: .TopSitesRemoveButtonAccessibilityLabel,
                                                 self.titleText.text ?? "")
+        closeButton.showsLargeContentViewer = true
+        closeButton.largeContentTitle = .TopSitesRemoveButtonLargeContentTitle
```

Comment:
I reused an existing accessibility label. Should I create a separate localized string for the `largeContentTitle`?